simple/rdm_pingpong.c

```diff
@@ -302,18 +302,9 @@ static int bind_ep_res(void)
 static int init_fabric(void)
 {
 	struct fi_info *fi;
-	char *node;
-	uint64_t flags = 0;
 	int ret;
 
-	if (opts.dst_addr) {
-		node = opts.dst_addr;
-	} else {
-		node = opts.src_addr;
-		flags = FI_SOURCE;
-	}
-
-	ret = fi_getinfo(FT_FIVERSION, node, opts.port, flags, &hints, &fi);
+	ret = fi_getinfo(FT_FIVERSION, opts.dst_addr, opts.dst_port, 0, &hints, &fi);
```

Comment:
The simple examples don't have  -s (source address) and -p option. For those, we can leave the code as it is for simplicity.